python/core/geometry/qgscurvepolygon.sip

```diff
@@ -57,6 +57,14 @@ class QgsCurvePolygon: public QgsSurface
     /** Removes ring. Exterior ring is 0, first interior ring 1, ...*/
     bool removeInteriorRing( int nr );
 
+    /**
+     * Removes the interior rings from the polygon. If the minimumRingArea
+     * parameter is specified then only rings smaller than this minimum
+     * area will be removed.
+     * @note added in QGIS 3.0
+     */
+    void removeInteriorRings( double minimumRingArea = -1 );
```

Comment:
@nyalldawson , I'm wondering whether it'd be useful to have a maximumRingArea parameter too. Say you want to simplify a feature to display nicely at a given zoomed-out level, you might want to remove rings that are below a maximumArea threshold.